content/docs/studio/index.md

```diff
@@ -1,11 +1,12 @@
 # Iterative Studio
 
 [Iterative Studio](https://studio.iterative.ai/) is a web application that you
-can access online or even host on-prem. It works with the data, metrics, and
+can access online or even host on-prem. It works with the data, metrics and
 hyperparameters that you add to your ML project repositories. Using the power of
-leading open-source tools DVC, [CML](https://cml.dev), and Git, it enables you
-to seamlessly manage data and models, run and track experiments, and visualize
-and share results.
+leading open-source tools [DVC](https://dvc.org/), [CML](https://cml.dev),
+[MLEM](https://mlem.ai/), [GTO](https://github.com/iterative/gto) and
```

Comment:
💅🏼 
```suggestion
[MLEM](https://mlem.ai/), [GTO](https://github.com/iterative/gto), and
```

Reply:
IMO [serial commas](https://www.thepunctuationguide.com/comma.html#thelistingcomma) are more than a 💅🏼 stylistic choice. Some times omitting it can cause confusing sentences. It's best to always include it in technical docs, which tend to be complicated -- so the clearest we can be, the better.

> No need to address this now, let's just keep in in mind for the future 🙂